.travis.yml

```
@@ -1,15 +1,19 @@
 language: python
+dist: focal
 python:
   - "3.6"      # current default Python on Travis CI
   - "3.7"
   - "3.8"
+  - "3.9"
 # command to install dependencies
 install:
+  - pip install --upgrade setuptools packaging wheel
   - pip install -r dev_requirements.txt
 # command to run tests
 script:
+  - python3 setup.py bdist_wheel
```

```suggestion
  - python3 setup.py sdist bdist_wheel
```

We need to keep the `sdist` build, as it get uploaded to pypi in the deploy stage